dynamodb_test.go

```diff
@@ -20,7 +19,7 @@ const (
 func TestIntegration(t *testing.T) {
 	ctx := context.Background()
 
-	container, err := RunContainer(ctx)
+	container, err := Run(ctx, "amazon/dynamodb-local:2.2.1")
```

Comment:
Maybe refactor the valid image+tag to a constant at the top of the file, or even in the non-test code, to make it easier to update in just one place ?

Reply:
SGTM, thanks!